backend/dataall/cdkproxy/stacks/policies/secretsmanager.py

```diff
@@ -0,0 +1,51 @@
+from .service_policy import ServicePolicy
+from aws_cdk import aws_iam
+
+
+class SecretsManager(ServicePolicy):
+    """
+    Class including all permissions needed to work with AWS Secrets Manager.
+    It allows data.all users to:
+    -
+    """
+
+    def get_statements(self):
+        statements = [
+            aws_iam.PolicyStatement(
+                #sid="SecretsReadAll",
+                effect=aws_iam.Effect.ALLOW,
+                actions=["secretsmanager:ListSecrets"],
+                resources=["*"],
```

Comment:
Just to clarify: we set resources * for ListSecrets, listQueues, and etc. because it raises errors is we try to get dataall the secret values, doesn't it?
I am interested if there is a way to get a dataall secret value (when we know the name of secret) without trying to list all secrets?